var/spack/repos/builtin/packages/gcc/package.py

```diff
@@ -337,11 +337,11 @@ class Gcc(AutotoolsPackage, GNUMirrorPackage):
     #   on XCode 12.5
     conflicts("+bootstrap", when="@:11.1 %apple-clang@12.0.5")
 
-    # aarch64/M1 is supported in GCC 11.3-12.2
+    # aarch64/M1 is supported in GCC 11.3-12.2 and 13.1
     conflicts(
-        "@:11.2,12.3:",
+        "@:11.2,12.3",
```

Comment:
The conflict change seems wrong, as it does not represent the `11.3-12.2` + `13.1` sets of versions. Can we turn this conflict into a `requires`?

Reply:
Thanks for getting to this—the PR was submitted before 13.2 was added. I'll see if 13.2 can be made to work in a similar way, and switch this over to a `requires`.

Reply:
Done. Can the review be finished?